commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -1299,10 +1299,9 @@ protected CommerceOrder startWorkflowInstance(
 		Map<String, Serializable> workflowContext = new HashMap<>();
 
 		return WorkflowHandlerRegistryUtil.startWorkflowInstance(
-			commerceOrder.getCompanyId(), commerceOrder.getScopeGroupId(),
-			userId, CommerceOrder.class.getName(),
-			commerceOrder.getCommerceOrderId(), commerceOrder, serviceContext,
-			workflowContext);
```

Comment:
@alecsloan This is not the right solution, The scope of the workflow must be related to the account

Reply:
So just delete the workflow by account group id?